app/controllers/games_controller.rb

```diff
@@ -25,14 +25,24 @@ def update
 		redirect_to game_path(@game)
 	end
 
-	def new
+  def concede
+    @game = Game.find(params[:id])
+    @black_player = User.find_by(id: @game.black_uid)
+    @white_player = User.find_by(id: @game.white_uid)
+    @current_player = current_user.id
+      if @black_player == @current_player
```

Comment:
I don't think this will work you are compairing a black_player to a current_player but current player is just an integer and black player is an object of class User.  Test this method more I think you will see it's broken